docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
+ACS-Engine and Kubernetes will need a service principal to create more resources such as Azure Load Balancers later once the cluster is running. It's safest to create one with access just to the resource group so that once your deployment is deleted, the service principal can't be used to make other changes in your subscription.
 
-11/16/2017  09:45 PM    <DIR>          .
-11/16/2017  09:45 PM    <DIR>          ..
-               0 File(s)              0 bytes
-               2 Dir(s)   5,368,709,120 bytes free
+```powershell
+# Get the group id
+$groupId = (az group show --resource-group <group name> --query id).Replace("""","")
 
+# Create the service principal
+$sp = az ad sp create-for-rbac --role="Contributor" --scopes=$groupId | ConvertFrom-JSON
 ```
 
-### Create Azure Disk workload
-This example is modified after https://github.com/andyzhangx/Demo/tree/master/windows/azuredisk/rs3
+### Create an acs-engine apimodel
 
-#### 1. Create an azure disk storage class
+Multiple samples are available at https://github.com/Azure/acs-engine/tree/master/examples/windows. This guide will use the simplest sample to deploy 1 Linux VM to run Kubernetes services, and 2 Windows nodes to run your Windows containers.
 
-##### option#1: k8s agent pool is based on blob disk VM
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/storageclass-azuredisk.yaml```
-
-##### option#2: k8s agent pool is based on managed disk VM
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/storageclass-azuredisk-managed.yaml```
-
-#### make sure storageclass is created successfully
-```
-kubectl get storageclass/azuredisk -o wide
+```powershell
+Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/Azure/acs-engine/master/examples/windows/kubernetes.json -OutFile kubernetes-windows.json
 ```
 
-#### 2. Create a pvc for azure disk
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/pvc-azuredisk.yaml```
+After downloading that file, you will need to
 
-#### make sure pvc is created successfully
-```
-kubectl get pvc/pvc-azuredisk -o wide
-```
+1. Set windowsProfile.adminUsername and adminPassword. Be sure to check the Azure Windows VM [username](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/faq?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json#what-are-the-username-requirements-when-creating-a-vm) and [password](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/faq?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json#what-are-the-password-requirements-when-creating-a-vm) requirements first.
+2. Set a unique name for masterProfile.dnsPrefix. This will be the first part of the domain name you'll use to manage the Kubernetes cluster later
+3. Set the ssh public key that will be used to log into the Linux VM
+4. Set the Azure service principal for the deployments

```

Comment:
They also have to choose disk size and Windows Version unless the kubernetes.json in the windows folder has been updated